string text             # the text to speak
string fallback_id      # the id of the system to fallback to if Ghost doesn't have a response